custom_components/goecharger_mqtt/number.py

```diff
@@ -43,10 +43,13 @@ def __init__(
 
     async def async_set_native_value(self, value: float) -> None:
         """Update the current value."""
-        if self.native_step == 1:
-            await mqtt.async_publish(self.hass, f"{self._topic}/set", int(value))
+        if self._topic.endswith("/dwo") and value == 0:
```

Comment:
I don't like this special case. Could you extend the `GoEChargerSensorEntityDescription` call by a new property which can be enabled per definition? This would be generic and reusable for other entities.

Reply:
Yes, you are right, it is done. I guess you meant to add it to `GoEChargerNumberEntityDescription`.